scripts/validate_docstrings.py

```diff
@@ -471,6 +471,45 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        """Return a parameter description as a list.
+
+        Each line of the parameter docstring is a string in the list.
+        """
+        i = list(self.doc_parameters).index(param)
```

Comment:
Why are you converting to a list here instead of looking up by key?

Reply:
Hi @WillAyd , I do this because `self.doc._parsed_data['Parameters']` returns a list of tuples, one for each parameter. I want to identify the position of the parameter.